network-store-integration-test/src/test/java/com/powsybl/network/store/integration/NetworkStoreIT.java

```diff
@@ -1626,6 +1626,42 @@ public void cgmesExtensionsTest() {
         }
     }
 
+    @Test
+    public void connectablesTest() {
+        try (NetworkStoreService service = createNetworkStoreService()) {
+            // import new network in the store
+            Network network = service.importNetwork(CgmesConformity1Catalog.miniNodeBreaker().dataSource());
+            service.flush(network);
+        }
```

Comment:
Perhaps test after and before flush ?
This would detect some regressions of `getAll` method
Currently `getSome` has a problem with this

Reply:
done